config/routes.rb

```diff
@@ -5,4 +5,13 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root "pages#home"
+  resources :pathways, only: %w[index show] do 
+    resources :challenges, only %w[index]
```

Comment:
If we display challenges belonging to a certain pathway, we can do on the 'pathway_show' page.
In this case, the 'challenges_index' page might not be needed.

Reply:
good point we'd just need a challenges show